scripts/build/artifactory.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I would change the name of this file. Currently it is named after the service that uses it but it would be better to name it after what it does, say `get_version_info.sh` or something like this.